src/components/ResourceTypeSelector/ResourceTypeSelector.js

```diff
@@ -41,21 +40,28 @@ const ResourceTypeSelector = ({
   selectResourceTypeAction,
   selectedResourceType,
 }) => (
-  <ScrollView style={styles.root} horizontal showsHorizontalScrollIndicator={false}>
-    {Object.entries(resourceTypeFilters).map(([resourceType, filterOpen]) => {
-      if (filterOpen) {
-        return (
-          <CategoryButton
-            key={resourceType}
-            resourceType={resourceType}
-            selectedResourceType={selectedResourceType}
-            selectResourceTypeAction={selectResourceTypeAction}
-          />
-        );
-      }
-      return null;
-    })}
-  </ScrollView>
+  <View>
+    <ScrollView
+      style={styles.root}
+      horizontal
+      showsHorizontalScrollIndicator={false}
+      contentContainerStyle={styles.contentContainerStyle}
+    >
+      {Object.entries(resourceTypeFilters).map(([resourceType, filterOpen]) => {
```

Comment:
is `filterOpen` mis-named?  

(eg, should it be: `isVisible`, `showType`, or somesuch?)

Reply:
`filterOpen` is also used in `src/components/FilterDrawer/FilterDrawer.js`, where it has a different meaning

Reply:
`filterOpen` is describing if this `resourceTypeFilter` is open. If it is "closed" or `false` then don't render the `resourceType` button.

For this component, I suppose we could abstract the filter value to an `isVisible` type flag because that's what it's effectively used for here.